tests/test_searcher.py

```diff
@@ -442,10 +452,54 @@ def test_sorting_legal_moves(
         board = init_board(fen_string)
         s = _init_searcher
 
-        legal_moves = s._ordered_moves(board, board.legal_moves)
+        legal_moves = s._ordered_moves(board, board.legal_moves, 0)
 
         num_moves = len(move_scores)
         for i, move in enumerate(legal_moves):
             mo = MvvLvaHeuristic()
-            score = mo.evaluate(board, move)
+            score = mo.evaluate(board, move, 0)
             assert score == move_scores[num_moves - i - 1]
+
+
+@pytest.mark.parametrize(
+    ("fen_string", "move_scores"),
+    [
+        ("k7/3q4/8/K7/2N5/8/8/8 w - - 1 34", [0, 0, 0, 0, 1, 0, 0, 0, 0, 0]),
```

Comment:
Where did you get this fen string from? 

[Very minor] Suggest using one of the fen string we have in the helper module? Easier to read and interpret

Reply:
I got it off a position I made. I feel it's more suited to the killer move setting - forking king/queen

Reply:
Suggest an inline comment??